agasc/tests/test_obs_status.py

```diff
@@ -434,3 +436,26 @@ def recreate_test_supplement():
     update_obs_status.update_obs_table(TEST_DATA_DIR / 'agasc_supplement.h5',
                                        obs_status_override,
                                        dry_run=False)
+
+
+def test_save_version(monkeypatch):
```

Comment:
Nice with the monkey patching!  I haven't really gone down this path before but it looks good.

Reply:
I think we should do more of that in unit testing so it does not depend on the actual SKA/data, and leave the use of SKA/data for integration testing. I think one can also use monkeypatch to generate the data files we would use (say, for cheta)